x/concentrated-liquidity/types/msgs_test.go

```diff
@@ -64,34 +64,6 @@ func TestMsgCreatePosition(t *testing.T) {
 			},
 			expectPass: false,
 		},
-		{
-			name: "lower tick < min tick",
-			msg: types.MsgCreatePosition{
-				PoolId:          1,
-				Sender:          addr1,
-				LowerTick:       types.MinTick - 1,
-				UpperTick:       1,
-				TokenDesired0:   sdk.NewCoin("stake", sdk.OneInt()),
-				TokenDesired1:   sdk.NewCoin("osmo", sdk.OneInt()),
-				TokenMinAmount0: sdk.OneInt(),
-				TokenMinAmount1: sdk.OneInt(),
-			},
-			expectPass: false,
-		},
-		{
-			name: "upper tick > max tick",
-			msg: types.MsgCreatePosition{
-				PoolId:          1,
-				Sender:          addr1,
-				LowerTick:       1,
-				UpperTick:       types.MaxTick + 1,
-				TokenDesired0:   sdk.NewCoin("stake", sdk.OneInt()),
-				TokenDesired1:   sdk.NewCoin("osmo", sdk.OneInt()),
-				TokenMinAmount0: sdk.OneInt(),
-				TokenMinAmount1: sdk.OneInt(),
-			},
-			expectPass: false,
-		},
```

Comment:
Just wondering - did the notion of min tick and max tick get removed?

Reply:
It didn't get removed, its just dependent on what the exponentAtPriceOne is. The only way I could feasibly make this work is hard coding it at the msg server level which doesnt feel correct. So right now it just gets checked on position creation / modification (non msg server level)